src/test/java/org/aludratest/service/jms/TestJmsActionImpl.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (C) 2015 Hamburg Sud and the contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.aludratest.service.jms;
+
+import org.aludratest.config.impl.SimplePreferences;
+import org.aludratest.service.jms.impl.JmsServiceImpl;
+import org.apache.activemq.broker.BrokerService;
+import org.apache.activemq.jndi.ActiveMQInitialContextFactory;
+import org.apache.commons.lang.StringUtils;
+import org.junit.*;
+
+import javax.jms.Message;
+import javax.jms.MessageListener;
+import javax.jms.TextMessage;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by ojurksch on 06.04.2016.
+ */
+public class TestJmsActionImpl {
```

Comment:
Please rename to JmsActionImplTest. We use TestXXX usually for test (e.g. mock) implementations.